.gitignore

```diff
@@ -3,6 +3,9 @@
 
 ### VisualStudioCode Patch ###
 
+### Intellij IDEA ###
+/.idea/
+
```

Comment:
I'm not familiar with this. I assume it won't be an issue, but a little explanation on why it's being added would have been nice.

Reply:
this is the settings file generated when using IntelliJ as a code editor. I will remove this.